client/src/panels/panel_declarations/finances/auth_exp_planned_spending/auth_exp_planned_spending.yaml

```diff
@@ -40,40 +40,35 @@ gov_auth_exp_planned_spending_body:
-    non utilisées au fin de l’exercice financier. En moyenne, les autorisations non utilisées représentaient **{{fmt_percentage1 hist_avg_tot_pct}}**
-    des autorisations.
+    TODO
     
     {{#if has_planned_spending}}Les dépenses {{fr_will_change_to plan_change "f" "p" ""}} **{{fmt_compact1_written gov_exp_planning_year_3}}** d'ici à {{last_planned_year}}. {{/if}}
 
 dept_auth_exp_planned_spending_body:
   transform: [handlebars, markdown]
   en: |
-    Over the last 5 years, **{{dept.name}}** has received average {{gl_tt "authorities" "AUTH"}} of **{{fmt_compact2_written dept_five_year_auth_average}}** while
+    Over the last 5 years, **{{dept.name}}** has received on average **{{fmt_compact2_written dept_five_year_auth_average}}** in {{gl_tt "authorized" "AUTH"}} spendings while
     {{gl_tt "actually spending" "BUD_EXP"}} an average of **{{fmt_compact2_written dept_five_year_exp_average}}**.
     
-    In **{{last_history_year}}**, **{{fmt_compact1_written last_year_lapse_amt}}** or **{{fmt_percentage1 last_year_lapse_pct}}** of approved
-    authorities were unspent at the end of the fiscal year. On average, unused authorities represented **{{fmt_percentage1 hist_avg_tot_pct}}** of authorities.
+    In **{{last_history_year}}**, **{{fmt_compact1_written last_year_lapse_amt}}** or **{{fmt_percentage1 last_year_lapse_pct}}** of authorized
+    spendings were unspent at the end of the fiscal year. On average, **{{fmt_percentage1 hist_avg_tot_pct}}** of authorized spendings go unused.
     
     {{#if has_planned_spending}}Spending is {{will_change_to plan_change ""}} **{{fmt_compact2_written dept_exp_planning_year_3}}** by {{last_planned_year}}. {{/if}}
   fr: |
-    Au cours des cinq dernières années, **{{dept.name}}** a reçu des {{gl_tt "autorisations" "AUTH"}} moyennes de **{{fmt_compact1_written dept_five_year_auth_average}}**
-    alors que les {{gl_tt "dépenses" "BUD_EXP"}} réelles s'élevaient en moyenne à **{{fmt_compact1_written dept_five_year_exp_average}}**.
-    
-    En {{last_history_year}}, **{{fmt_compact1_written last_year_lapse_amt}}** ou **{{fmt_percentage1 hist_avg_tot_pct}}** des autorisations approuvées étaient
-    non utilisées au fin de l’exercice financier. En moyenne, les autorisations non utilisées représentaient **{{fmt_percentage1 hist_avg_tot_pct}}**
-    des autorisations.
+    TODO
     
     {{#if has_planned_spending}}Les dépenses {{fr_will_change_to plan_change "f" "p" ""}} **{{fmt_compact2_written dept_exp_planning_year_3}}** d'ici à {{last_planned_year}}. {{/if}}
 
 budgetary_expenditures:
-  en: Budgetary Expenditures
-  fr: Dépenses budgétaires
+  en: Actual Spending
+  fr: Dépenses Réelles
+
+actual_spending_footnote:
+  transform: [handlebars]
+  en: '"Actual Spending" numbers are technically {{gl_tt "Budgetary Expenditures" "BUD_EXP"}}.'
+  fr: 'Les nombres de "Dépenses Réelles" sont techniquement {{gl_tt "Dépenses budgétaires" "BUD_EXP"}}.'
```

Comment:
Maybe "Here, "actual spending" refers to budgetary expenditure values"?

Something about the tone of "technically" feels off, maybe because it feels like we're correcting ourselves rather than clarifying.